### ISSUE TITLE
Training notebook not working

### ISSUE THREAD

**Issue by kingafy** [JohnGiorgi/DeCLUTR#200]
File ‘preprocess_wikitext_103.py’ already there; not retrieving.

```bash
2021-04-06 09:59:13.361932: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
Traceback (most recent call last):
  File "preprocess_wikitext_103.py", line 117, in <module>
    typer.run(main)
  File "/usr/local/lib/python3.7/dist-packages/typer/main.py", line 859, in run
    app()
  File "/usr/local/lib/python3.7/dist-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/typer/main.py", line 239, in get_command
    click_command = get_command_from_info(typer_instance.registered_commands[0])
  File "/usr/local/lib/python3.7/dist-packages/typer/main.py", line 423, in get_command_from_info
    ) = get_params_convertors_ctx_param_name_from_function(command_info.callback)
  File "/usr/local/lib/python3.7/dist-packages/typer/main.py", line 404, in get_params_convertors_ctx_param_name_from_function
    click_param, convertor = get_click_param(param)
  File "/usr/local/lib/python3.7/dist-packages/typer/main.py", line 635, in get_click_param
    assert len(types) == 1, "Typer Currently doesn't support Union types"
AssertionError: Typer Currently doesn't support Union types
```

Also I want to experiment this on a non-wiki dataset.. I have got input txt file .What shoulf be the output txt format so that it can be used for training.

**Comment by JohnGiorgi** [comment]
Hi @kingafy 

Ah I see, the bug is here

```
AssertionError: Typer Currently doesn't support Union types
```

Weird because this use to work without error. Anyways, thanks for the heads up. I fixed it in #201. Try running a fresh copy of the notebook?

> Also I want to experiment this on a non-wiki dataset.. I have got input txt file .What shoulf be the output txt format so that it can be used for training.

An input text file is all you need. It just needs to contain one document per line. See [Preparing a Dataset](https://github.com/JohnGiorgi/DeCLUTR#preparing-a-dataset) and [Training](https://github.com/JohnGiorgi/DeCLUTR#training) in the readme and let me know if you still have questions.

**Comment by JohnGiorgi** [comment]
Closing this as I believe the issue is solved.
